plugin.rb

```diff
@@ -20,7 +20,10 @@
 Discourse::Application.routes.append do
   mount ::DiscourseAssign::Engine, at: "/assign"
   get "topics/private-messages-assigned/:username" => "list#private_messages_assigned", as: "topics_private_messages_assigned", constraints: { username: ::RouteFormat.username }
-  get "topics/messages-assigned/:username" => "list#messages_assigned", as: "topics_messages_assigned", constraints: { username: ::RouteFormat.username }
+  get "/topics/messages-assigned/:username" => "list#messages_assigned", constraints: { username: ::RouteFormat.username }
+  get "/topics/group-messages-assigned/:groupname" => "list#group_messages_assigned", constraints: { username: ::RouteFormat.username }
```

Comment:
group-messages-assigned is kind of confusing, because 'messages' implies 'private messages'.

Please can we rename it to `group-topics-assigned`? And also rename the matching method names.

The old `messages-assigned` route can stay for now, we'll look at changing it in the future.